superset/dataframe.py

```diff
@@ -95,7 +141,7 @@ def agg_func(cls, dtype, column_name):
         # consider checking for key substring too.
         if cls.is_id(column_name):
             return 'count_distinct'
-        if (issubclass(dtype.type, np.generic) and
+        if (hasattr(dtype, 'tupe') and issubclass(dtype.type, np.generic) and
```

Comment:
Is this a typo? `tuple`?

Reply:
Typo, the intent was to check that `dtype.type` actually exists. I'm not 100% sure whether I actually hit a missing attr or if it was preventative.